packages/typespec-ts/test/modularIntegration/generated/type/property/additional-properties/src/utils/serializeUtil.ts

```diff
@@ -1,27 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license.
-
-import {
-  WidgetData1 as WidgetData1Rest,
-  WidgetData as WidgetDataRest,
-} from "../rest/index.js";
-import { WidgetData1, WidgetData } from "../models/models.js";
-
-/** serialize function for WidgetData1 */
-function serializeWidgetData1(obj: WidgetData1): WidgetData1Rest {
```

Comment:
I don't think we should remove this? As it should be discriminated union or polymorhic base?

Reply:
This is now handled in the models.ts file